components/cms/Toolbar.tsx

```diff
@@ -27,23 +53,39 @@ export const Toolbar = styled(({ ...styleProps }) => {
           <CreateContentMenu />
         </Create>
         <Github>
```

Comment:
Maybe out of scope for this PR, but this doesn't seem like it needs to be scoped to Github